src/cli.ts

```diff
@@ -22,7 +27,7 @@ const cli = meow(`
 		context: {
 			alias: 'c',
 			type: 'string',
-			isRequired: true
+			isRequired: false
```

Comment:
context needs rename to `file`, as an alternative to the context you set with `context` command, right?

Reply:
I was thinking that CLI will load the context file, but `--context` could override the existing context for that command execution.

Reply:
ok, this makes sense

adding `--file` can be done separately

> I was thinking that CLI will load the context file

it would also be awesome that in case context is not there, the CLI also check if in the execution path there are files like `asyncapi.yml`, `asyncapi.yaml` or `asyncapi.json` and load them automatically.

Reply:
ok, so you mean, in the working directory. This would be awesome. 

So 
- First check if there is any `asyncapi.yml`, `asyncapi.yaml`, `asyncapi.json` in the working directory 
- If not then check globally for the `.asyncapi` file

Reply:
I'm not really sure about the priority here, no strong opinion

> First check if there is any asyncapi.yml, asyncapi.yaml, asyncapi.json in the working directory

this could be first to promote proper filename pattern, but we should also clearly log information about what file is used by the CLI

> If not then check globally for the .asyncapi file

this could also be first as it could be confusing for a user that he set a context but a different file is read.

better DX would be I think if we follow your priority order